CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.9.0-dev
```

Comment:
As a new feature, perhaps `4.9.0`? You may just want to merge this w/ https://github.com/dart-lang/code_builder/pull/439 and rev directly to `4.9.0` (instead of `4.9.0-dev`).